### PR TITLE
	new file:   dmbiolib/meta.yaml

### PR DESCRIPTION
**Pull request by damienmarsic**
Describe your pull request here

Add dmbiolib
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by damienmarsic** [comment]
OK

**Comment by damienmarsic** [comment]
@BiocondaBot please add label

**Comment by bhaller** [comment]
Please remove me from your review list.  I seem to have been added somewhere, but I barely understand how to update my own project; I have no idea how to review changes for other projects.

**Comment by bsiranosian** [comment]
Not sure why I'm a reviewer on this either

File: recipes/dmbiolib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set name = "dmbiolib" %}
+{% set version = "0.3.7" %}
+
+package:
+  name: "{{ name|lower }}"
```
Comment (holtgrewe) [inline recipes/dmbiolib/meta.yaml:5]: `|lower` is not necessary here

File: recipes/dmbiolib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - python -m dmbiolib -v
+
+about:
+  home: "https://github.com/damienmarsic/dmbiolib"
+  license: "GNU General Public v3 (GPLv3)"
+  license_family: GPL3
+  license_file: LICENSE
+  summary: "Library of Python functions for bioinformatics"
+  doc_url: https//dmbiolib.readthedocs.io
+  dev_url:
```
Comment (holtgrewe) [inline recipes/dmbiolib/meta.yaml:37]: remove key?

**Review by holtgrewe — CHANGES_REQUESTED** [review]

File: recipes/dmbiolib/meta.yaml (hunk 2 above)
Comment (damienmarsic) [reply recipes/dmbiolib/meta.yaml]: Thanks for the suggestions. I made the requested changes.

**Review by holtgrewe — APPROVED** [review]
LGTM

**Comment by damienmarsic** [comment]
Thanks for the approval :)

**Pull request merged**